website/docs/utilities/popover-primitive/partials/code/component-api.md

```diff
@@ -0,0 +1,123 @@
+## Component API
+
+<Doc::ComponentApi as |C|>
+  <C.Property @name="isOpen" @type="boolean" @default="false" @values={{array "true" "false"}}>
+    Controls if the popover should be rendered initially opened.
+    <br />
+    <br />
+    _Notice: in this case the popover can't be dismissed via `esc` or `click outside` until the end user has interacted with it (it's in a "manual" state)._
+  </C.Property>
+  <C.Property @name="enableSoftEvents" @type="boolean" @default="false" @values={{array "true" "false"}}>
+    Assigns "soft" events listeners (`mouseEnter/Leave` + `focusIn/Out`) to the toggle, to control the visibility of the popover content.
```

Comment:
Question: where does the term "soft event" come from?

Reply:
We came up with this term to refer to non-click events. Do you have suggestions for an alternative term?

Reply:
I'm thinking about it now, I won't object if I can't figure out something else to propose. Right now it feels technically inaccurate so I will try.

Reply:
What do you think about `enableIntentEvents`?  Intent is a more technically accurate term for what we're going for, WDYT?

Reply:
I think "soft" makes decent sense since clicking is a more active "hard" interaction so hovering & focusing are less active so "softer"...